yarn-project/noir-contracts/contracts/docs_example_contract/src/main.nr

```diff
@@ -45,31 +45,6 @@ contract DocsExample {
         imm_singleton: ImmutableSingleton<CardNote>,
     }
 
-    impl Storage {
-        fn init(context: Context) -> Self {
-            Storage {
-                leader: PublicState::new(
-                    context,
-                    1
-                ),
-                // docs:start:start_vars_singleton
-                legendary_card: Singleton::new(context, 2),
-                // docs:end:start_vars_singleton
```

Comment:
when deleting these, be careful to modify the docs!

Reply:
Done!